app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.length)
```

Comment:
- [ ] Optional suggestion

I would recommend doing something like this to count the posts.

```
def  update_comments_counter
  post.increment!(:comments_counter)
end
```